tests/cephfs/CEPH-10625_11225.py

```diff
@@ -297,31 +297,16 @@ def run(ceph_cluster, **kw):
                 )
         for num in range(0, 5):
             for client in client_info["fuse_clients"]:
-                client.exec_command(
-                    cmd="sudo crefi %s'%s' --fop create -t %s "
-                    "--multi -b 10 -d 10 -n 10 -T 10 "
-                    "--random --min=1K --max=%dK"
-                    % (client_info["mounting_dir"], dir_name, "text", 5),
-                    long_running=True,
-                )
-                for i in range(0, 6):
-                    ops = ["create", "rename", "chmod", "chown", "chgrp", "setxattr"]
-                    rand_ops = random.choice(ops)
```

Comment:
`range` was used to enable random iteration of `ops` however in the new code, this is undone. If the intent was to undone random selection of operations... then you could remove `range` also and iterate through operations.